lib/ocamlorg_web/graphql.ml

```diff
@@ -0,0 +1,330 @@
+open Ocamlorg
+
+type package_dep =
+  { name : string
+  ; constraints : string option
+  }
+
+type package_owner =
```

Comment:
I think we can do a similar thing here that we did offline with `Package.Info.url` and reuse the `Opam_user` type. If you look at your `get_owners` function below it is simply rewrapping each field into this new type so since there's no converting of values, I think it makes sense to reuse the `Opam_user` type. 

So the schema for the user would replace the resolvers to directly use `Opam_user` for example the name field would become.

```ocaml
field
    "name"
    ~typ:(non_null string)
    ~args:Arg.[]
    ~resolve:(fun _ user -> user.Opam_user.name)
```